src/universal/utils/makeCheckinGreeting.js

```diff
@@ -27,7 +72,8 @@ const questions = [
   'What are you cooking next',
   'Learn any new slang lately',
```

Comment:
We've got good data the the slang question is hard for folks/too young. I think we should nuke it.